core/scripts/functions/USAGE.md

```diff
@@ -25,7 +25,7 @@ This sub-folder contains two `toml` templates that will be processed with `gener
 2. Run the tool: `go run .` to see available commands.
 
 Initial set of commands:
-* `generate-ocr2config` generates `OCR2DROracleConfig.json` that is consumed by contracts tooling.
+* `generate-ocr2config` generates `OCR2Config.json` that is consumed by contracts tooling.
```

Comment:
maybe FunctionsOCR2Config.json?

Reply:
I thought the same, but then realized it is just OCR2 thing. Internally this config has no Functions context.

Reply:
Alright, will name it as suggested!

Reply:
Done